wg-code-quality/wg-charter.md

```diff
@@ -0,0 +1,64 @@
+# WG Code-Quality Charter
+
+## Scope
+
+The Code-Quality Working Group has two goals:
+
+* (short-term) **focus on the improvement of code quality** and
+* (long-term) **create an inter-SIG process for identifying, agreeing on and addressing code quality issues**
+
+The long-term goal is achieved when the process is implemented and executed as required.
+
+## In scope
+
+The main objective of this effort is to continuously improve the quality of the project codebase.
```

Comment:
Since the project is quite big, is there an idea of where to start?

Reply:
The charter is focused on *what* is needed rather on *how*.
The definition of the scope just provides the understanding of what the WG is considering under its influence/scope.

As @dhiller pointed out, we will discuss how to do things and which areas to invest first in our bi-weekly meetings and through regular communications.